src/CMakeLists.txt

```diff
@@ -58,7 +58,7 @@ Windows systems.")
 set(WITH_LIDAR ON CACHE BOOL "Enable/Disable the USB LiDAR interface.")
 set(WITH_TESTS ON CACHE BOOL "Enable/Disable the tests.")
 set(WORLD_INTERFACE "REAL" CACHE STRING "The world interface implementation to use.")
-set(AVAILABLE_WORLD_INTERFACES "REAL" "SIMULATOR" "NO-OP")
+set(AVAILABLE_WORLD_INTERFACES "REAL" "SIMULATOR" "NO-OP" "SIMULATOR3D")
```

Comment:
Yeah, `SIMULATOR` refers to the old planar simulator until we deprecate it.

After we deprecate it, do you think we should rename the new simulator to `SIMULATOR` or leave it as `SIMULATOR3D`? An alternative to leaving the old sim as `SIMULATOR` would be to rename it to `SIMULATOR2D` or something, but I'm n to sure if that's super relevant considering we're gonna remove it asap.

Reply:
I think it probably would be best to rename it `SIMULATOR` since there's no need to distinguish between 3D and 2D anymore, but I don't really have a strong preference.